Library/Homebrew/livecheck.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Livecheck
+  def initialize
+    @skipped = false
+    @skip_msg = nil
+    @url = nil
+  end
+
+  def skip(skip_msg = nil)
+    @skipped = true
+    @skip_msg = skip_msg
+  end
+
+  def url(val = nil)
```

Comment:
This is just my own perspective but it may be better for us to allow this to error if `url` is in a `livecheck` block and no argument is provided. That is to say, a default value doesn't seem appropriate here, as a passed URL should be required.

What does everyone else think about this?

Reply:
The default value is used in these sort of DSLs to allow the use of both `livecheck.url` and `livecheck do; url "$URL; end` i.e. this method is both a getter and a setter.

Reply:
That would mean it's best to remove the `ArgumentError` I've raised when the value is blank, so that it can function as both a getter and a setter?

Reply:
> That would mean it's best to remove the `ArgumentError` I've raised when the value is blank, so that it can function as both a getter and a setter?

Yup 👍